src/components/form/templates/array-field-simple-item.ts

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { ArrayFieldItem } from './types';
+
+export class SimpleItemTemplate extends React.Component {
+    private props: { item: ArrayFieldItem; index: number };
+    private refs: { removeButton: any; moveUpButton: any; moveDownButton: any };
+
+    private removeHandler: (event: any) => void;
+    private moveUpHandler: (event: any) => void;
+    private moveDownHandler: (event: any) => void;
+
+    public componentDidMount() {
+        const { item, index } = this.props;
+        const removeButton: HTMLLimelButtonElement = this.refs.removeButton;
+        this.removeHandler = item.onDropIndexClick(index);
+        removeButton.addEventListener('click', this.removeHandler);
+
+        const upButton: HTMLLimelButtonElement = this.refs.moveUpButton;
+        this.moveUpHandler = item.onReorderClick(index, index - 1);
+        upButton.addEventListener('click', this.moveUpHandler);
+
+        const downButton: HTMLLimelButtonElement = this.refs.moveDownButton;
+        this.moveDownHandler = item.onReorderClick(index, index + 1);
+        downButton.addEventListener('click', this.moveDownHandler);
+    }
+
+    public componentWillUnmount() {
+        const removeButton: HTMLLimelButtonElement = this.refs.removeButton;
+        removeButton.removeEventListener('click', this.removeHandler);
+
+        const upButton: HTMLLimelButtonElement = this.refs.moveUpButton;
+        upButton.removeEventListener('click', this.moveUpHandler);
+
+        const downButton: HTMLLimelButtonElement = this.refs.moveDownButton;
+        downButton.removeEventListener('click', this.moveDownHandler);
+    }
+
+    public render() {
+        const { item } = this.props;
+
+        return React.createElement(
+            'div',
+            {
+                className: 'limel-form-array-item--simple',
+            },
+            this.props.item.children,
+            this.renderRemoveButton(item),
+            this.renderMoveUpButton(item),
+            this.renderMoveDownButton(item)
+        );
+    }
+
+    private renderRemoveButton(item: ArrayFieldItem) {
+        // tslint:disable-next-line:no-duplicate-string
+        return React.createElement('limel-button', {
+            icon: 'trash',
+            disabled: !item.hasRemove,
+            ref: 'removeButton',
+        });
```

Comment:
I tried using `limel-icon-button` instead, and I like the result much better:

![image](https://user-images.githubusercontent.com/154725/73344650-0299bc00-4283-11ea-8a1e-4f954d35562a.png)

We should preferably supply labels for the buttons as well though (they won't be visible when using `limel-icon-button`), as those will be used by assistive tech, and we're going to have to move towards assistive tech compatibility anyway. But I guess that would mean either adding translation functionality to lime-elements, or requiring translated strings be input by the consuming client… So, I guess that'll be a matter for a future update regardless… 😕

Reply:
Yeah I tried them both and just went with one of them for now as I didn't want to decide this by myself :smile: Let's have a discussion during the meeting this afternoon. 
I'm not sure I like that these are bigger, but they have less margin to the left/right which I think is good :slightly_smiling_face: 
Also, there might be other icons we can use. It also require icons8 at the moment, which is not shipped with lime elements which might be an issue

Reply:
I didn't immediately like the bigger icons either. They are easier to see, but the are a bit big in comparison to the text. Although, I think we should maybe use slightly bigger text in the headers for collapsible-section…

That the regular button can't have a label for assistive tech without also visibly showing that label suggests to me that the icon-button is the right component to use for buttons with no text.

Regarding icons8, I agree that it's not ideal. I had a question from Fredrik the other day, if there was any way for an add-on to supply its own icon to a limel-button, and there isn't at the moment, but it would be really nice if, apart from a name of an icon, the consumer could also supply either a url for where to load the icon from, or the image-data itself. For a url, we could simply look at the first character of the `icon` string, and if it's a slash, we load the image from that url, instead of the normal way. I'm pretty sure image data has some known prefix as well, which we could detect. That way, we would only need to make changes to limel-icon, not to all the components using it. And then we could embed some icons that we are allowed to use, like I did with the chevron for collapsible-section.